utils/cliutils/commandsflags.go

```diff
@@ -1085,6 +1087,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Force,
 		Usage: "[Default: false] Set to true to allow config transfer to a non-empty Artifactory server.` `",
 	},
+	Merge: cli.BoolFlag{
+		Name:  Merge,
+		Usage: "[Default: false] Set to true to merge source in target.` `",
```

Comment:
Set to true to merge source in target.
-->
Set to true to merge Projects and Repositories from the source instance to the target